fdbcli/ExcludeCommand.actor.cpp

```diff
@@ -350,13 +357,12 @@ ACTOR Future<bool> excludeCommandActor(Reference<IDatabase> db, std::vector<Stri
 						printf("        Do not include the `:tls' suffix when naming a process\n");
 					return true;
 				}
-				exclusionVector.push_back(a);
 				exclusionSet.insert(a);
 				exclusionAddresses.push_back(a);
 			}
 		}
 
-		if (exclusionAddresses.empty() && exclusionLocalities.empty()) {
+		if (exclusionSet.empty()) {
```

Comment:
This changes the behaviour of the exclude command a bit. Before this change the exclude command would not complain if a locality was not matching any process. With the changes in this PR if no servers are targeted by a locality the exclude command will fail with:

```text
ERROR: At least one valid network endpoint address or a locality is not provided
```

I believe it's better to fail if a locality is not targeting any processes, e.g. to better catch cases where someone mistyped a locality.